Processes/How-To-Approve-and-Signoff-Content-Before-Publication-to-the-EOS-Documentation-Portal.md

```diff
@@ -0,0 +1,9 @@
+## How To Approve and Signoff Content before Publication to the EOS Documentation Portal
```

Comment:
I guess I'm not following this step in the process. We want to encourage contributors to directly submit their own PRs. That's how contributions are made in github, and how authorship is retained and confirmed in the commit history. If we submit the PR for them, we are claiming authorship. I'll stop my review here until this is clarified. @nsjames